roles/content/handlers/main.yml

```diff
@@ -1,14 +1,40 @@
 ---
 
-- name: install fxa-content-server dependencies
+# To avoid breaking tests by clobbering `dist` from underneath the running
+# content server, we copy the current working directory to a temporary
+# location, do the npm and grunt changes in the copy, and then swap the copy
+# into place (and restart).
+
+# there shouldn't be anything leftover but just in case
+- name: begin install fxa-content-server by removing any leftover in /var/tmp
+  sudo: true
+  sudo_user: app
+  command: rm -rf /var/tmp/fxa-content-server
+
+- name: begin install fxa-content-server dependencies with a copy
+  sudo: true
+  sudo_user: app
+  command: cp -rpf /data/fxa-content-server /var/tmp
+
+- name: install fxa-content-server dependencies 
   sudo: true
   sudo_user: app
-  command: npm install --production chdir=/data/fxa-content-server
+  command: npm install --production chdir=/var/tmp/fxa-content-server
 
 - name: build fxa-content-server assets
   sudo: true
   sudo_user: app
-  command: grunt build chdir=/data/fxa-content-server
+  command: grunt build chdir=/var/tmp/fxa-content-server
```

Comment:
it will convert the urls as part of grunt build, i was just wondering if the cdn update takes time to process new files in some way. let's have a quick chat in the meeting.